CompetitionTemplate/src/competition/subsystems/drive/commands/UpdatePoseCommand.java

```diff
@@ -30,6 +30,7 @@ public void initialize() {
     public void execute() {
         pose.updateCurrentHeading();
         defenseTraversal.measureState(Math.abs(pose.getRobotPitch()));
+        pose.updateRangefinders();
```

Comment:
Might be cleaner to just have a general `updateSensorState` on the pose subsystem and call that from here?